server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java

```diff
@@ -2,6 +2,24 @@
 
 import reactor.netty.http.server.HttpServer;
 
+/**
+ * Implement this interface in order to configure the server to your own needs
+ */
 public interface ReactorServerConfigurer {
+    /**
+     * Make custom configurations to the httpServer
+     * @param httpServer the server to be configured
+     * @return the configured server.
+     */
     HttpServer configure(HttpServer httpServer);
+
+    /**
+     * Returns which prio this configurer has.
+     *
+     * prio 1 will be executed before prio 2
```

Comment:
Which means that prio 2 will overwrite changes of prio 1 and therefore wins. No way to make things like this easy and intutive to understand... No action requested ;-)

Reply:
🧠